content/docs/warehouses/bigquery.md

```diff
@@ -94,6 +94,34 @@ config {
 
 You can configure separate project-ids for development and production in your `environment.json` file. The process is described on [this page](/dataform-web/scheduling/environments#example-use-separate-databases-for-development-and-production-data).
 
+### Optimizing partitioned incremental tables for BigQuery
+
+When creating incremental tables from partitioned tables in BigQuery, some extra care needs to be taken to avoid full table scans, as BigQuery can't optimize where statements that are computed from inline select statements. To fix this, values used in the where clause should be moved to a `pre_operations` block and saved into a variable using [BigQuery scripting](https://cloud.google.com/bigquery/docs/reference/standard-sql/scripting).
```

Comment:
```suggestion
When creating incremental tables from partitioned tables in BigQuery, some extra care needs to be taken to avoid full table scans, as BigQuery can't optimize `where` statements that are computed from inline select statements. To work around this, values used in the where clause should be moved to a `pre_operations` block and saved into a variable using [BigQuery scripting](https://cloud.google.com/bigquery/docs/reference/standard-sql/scripting).
```